Reconstruction/doc/RecCalorimeter.md

```diff
@@ -1,7 +1,7 @@
 RecCalorimeter package
 ===
 
-Information about calorimeter reconstruction software within FCCSW. The software is designed for ECAL, but should be general enough to be used for other calorimeters.
+Information about calorimeter reconstruction software within FCCSW. The software being tested using ECAL, but should be general enough to be used for other calorimeters. Let us know if you have any problems or questions.
```

Comment:
Typo in the second sentence, should be: "The software *is* being.."
Let us know -> add names or mailing list?